recipes/tinyexr/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "cci.20200811":
+    url: "https://github.com/syoyo/tinyexr/archive/cf8550f1b8b9f5f79f02df810c885ed2a2b578f9.tar.gz"
```

Comment:
There are official releases and project seems be still active. The last release is from 2016. Please, ask for a new tag to the upstream

Reply:
Master branch can be considered stable https://github.com/syoyo/tinyexr/issues/149#issuecomment-689336326

Reply:
Now we got a `1.0.0` release 😄

Reply:
Thank you people for asking to the author!